File: src/test/java/com/datarepublic/testexercise/ebayframework/SearchItemWidget.java
package com.datarepublic.testexercise.ebayframework;

import org.openqa.selenium.*;

public class SearchItemWidget extends BaseWidget
{   
    public SearchItemWidget(WebElement elem)
    {
        super(elem);
    }
    
    private final String itemLinkCss = ".lvtitle > a";
    
    void openItem()
    {
        boundingElem.findElement(By.cssSelector(itemLinkCss)).click();
    }
}

File: net/TestExerciseEbay/EbayPOMFramework/SearchItemWidget.cs
using System;
using OpenQA.Selenium;

namespace TestExercise.EbayPOMFramework
{
    public class SearchItemWidget : BaseWidget
    {
        public SearchItemWidget(IWebElement elem) : base(elem) { }

        private const string itemLinkCss = ".lvtitle > a";

        public void OpenItem()
        {
            boundingElem.FindElement(By.CssSelector(itemLinkCss)).Click();
        }
    }
}